docs/user-guide/tls-termination.rst

```diff
@@ -0,0 +1,223 @@
+=======================
+TLS Termination and DNS
+=======================
+
+This section describes how to expose a service via TLS to the internet.
+
+Expose your app
+===============
+
+Let's deploy a simple web server to test that our TLS termination works.
+Submit the following ``yaml`` files to your cluster.
+
+Create the deployment
+---------------------
+
+.. code-block:: yaml
+
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: nginx
+    spec:
+      replicas: 2
+      template:
+        metadata:
+          labels:
+            app: nginx
+        spec:
+          containers:
+          - name: nginx
+            image: nginx
+            ports:
+            - containerPort: 80
+
+This creates a deployment called ``nginx`` that will ensure to run two copies
+of the nginx image from dockerhub listening on port 80.
+
+Make sure your pods are running.
+
+.. code-block:: bash
+
+    $ kubectl get pods
+    NAME                     READY     STATUS    RESTARTS   AGE
+    nginx-1447934386-iblb3   1/1       Running   0          7m
+    nginx-1447934386-jj559   1/1       Running   0          7m
+
+Create the service
```

Comment:
As I understand, creating a service first will help kubernetes distribute the pods over several nodes http://kubernetes.io/docs/user-guide/config-best-practices/#services.

For this reason it might make sense to talk about the service first, to teach people about this approach.